apps/teams/context_processors.py

```diff
@@ -1,4 +1,33 @@
+from django.urls import reverse
+
+
 def team(request):
     return {
         "team": getattr(request, "team", None),
+        "notices": get_team_notices(request),
     }
+
+
+def get_team_notices(request):
+    team = getattr(request, "team", None)
+    if not team:
+        return []
+
+    return filter(None, [notice(request, team) for notice in NOTICES])
```

Comment:
Why the `filter(None, ...)`? Wouldn't this just return the iterable?

Reply:
It removes any that are 'falsy'

Reply:
Cool, did not know that!